modules/meeting/app/models/meeting_section.rb

```diff
@@ -42,6 +42,12 @@ class MeetingSection < ApplicationRecord
 
   scope :backlog, -> { where(backlog: true) }
 
+  def title
+    return read_attribute(:title) unless backlog?
```

Comment:
You can simply use `super` to call the default implementation of title

```suggestion
    return super unless backlog?
```